.github/workflows/unit_test.yml

```diff
@@ -43,10 +44,9 @@ jobs:
 
     - name: Run tests
       env:
-        JAVA_OPTS: -Xmx4G -Xms4G -Xss2M
+        JAVA_OPTS: -Xmx4G -Xms4G -Xss2M -Duser.timezone=UTC -Denv.type=test -Dmysql.host=localhost -Dmysql.port=3307
```

Comment:
Hmm... all the test failures have to do with the created date being off from the expected value it seems. Could it be this time zone flag?

Reply:
it's not timezone...for some reason, one timestamp is more precise, see 
```
org.scalatest.exceptions.TestFailedException: Some(DataprocInstance(DataprocInstanceKey(dsp-leo-test,ZoneName(my-zone),InstanceName(master-instance)),12345,Running,Some(IP(1.2.3.4)),Master,2022-04-12T16:12:06.553181Z)) did not equal Some(DataprocInstance(DataprocInstanceKey(dsp-leo-test,ZoneName(my-zone),InstanceName(master-instance)),12345,Running,Some(IP(1.2.3.4)),Master,2022-04-12T16:12:06.553180524Z))
```